README.md

```diff
@@ -61,10 +64,18 @@ res3: String = URL is invalid: no protocol: https//www.github.com
 
 scala> t2.toEither
 res4: Either[Throwable,java.net.URL] = Left(java.net.MalformedURLException: no protocol: https//www.github.com)
+
+scala> val intToBytes = 123456789.toByteArray
+intToBytes: Array[Byte] = Array(7, 91, -51, 21)
+
+scala> val longToBase64 = 123456789L.toBase64
+longToBase64: String = AAAAAAdbzRU
 ```
 
 #### Release Notes
 
+Version `0.13` (Dec 17) include Int/Long/Double toByteArray and toBase64 operations and is built against cats `1.0.0-RC1`
```

Comment:
Isn't it `RC2` ?

Reply:
sorry that change was picked up by mistake.